tests/integration-e2e/test_cli_opts.py

```diff
@@ -39,3 +40,36 @@ def test_cli_disable_reports(tmp_path):
     for name in sc.list_report_names:
         report_path = Path(sc.report(name).file_name)
         assert report_path.is_file(), f"File '{report_path}' not found."
+
+@pytest.mark.parametrize(
+    "create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "reports": {
+                "soma_v": {
+                    "type": "compartment",
+                    "variable_name": "v",
+                    "sections": "soma",
+                    "dt": 0.1,
+                    "start_time": 0.0,
+                    "end_time": 40.0
+                }
+           },
+        }
+    }
+], indirect=True,
+)
+def test_cli_report_buff_size(create_tmp_simulation_config_file):
```

Comment:
Can you also add a integration test for CORENEURON?

Reply:
Can I do this in a loop? I don't see anymore in the existing tests to switch between the two modes in a single test. Or should I simply do two functions, one for CoreNEURON, one for NEURON?

Reply:
answer just above :D